metadata/metadata.go

```diff
@@ -49,55 +43,13 @@ func (m *Client) GetVersion() (string, error) {
 	return m.MetadataClient.GetVersion()
 }
 
-// GetCronSchedules returns a map of schedules with ContainerUUID as a key
-func (m *Client) GetCronSchedules() (*model.Schedules, error) {
-	schedules := *m.Schedules
-	services, err := m.MetadataClient.GetServices()
-
-	if err != nil {
-		logrus.Infof("Error reading services %v", err)
-		return &schedules, err
-	}
-
-	markScheduleForCleanup(m.Schedules)
-
-	for _, service := range services {
-		cronExpression, ok := service.Labels[m.CronLabelName]
-		if !ok {
-			continue
-		}
-		containerUUIDs, err := m.getCronContainers(service)
-		if err != nil {
-			continue
-		}
-
-		for _, containerUUID := range containerUUIDs {
-			existingSchedule, ok := schedules[containerUUID]
-			// we already have schedule for this container
-			if ok {
-				// do not cleanup
-				existingSchedule.ToCleanup = false
-
-				logrus.WithFields(logrus.Fields{
-					"uuid":           containerUUID,
-					"cronExpression": cronExpression,
-				}).Debugf("already have container")
-
-				continue
-			}
-			//label exists, configure schedule
-			schedule := model.NewSchedule()
-
-			schedule.CronExpression = cronExpression
-			schedule.ContainerUUID = containerUUID
-			schedules[containerUUID] = schedule
-		}
-	}
-
-	return &schedules, nil
+// GetServices returns the services from the metadata client
+func (m *Client) GetServices() ([]metadata.Service, error) {
```

Comment:
It is still being called from here to get the original list of services with the cron label.
https://github.com/jmloeffler/rancher-cron/blob/master/scheduler/scheduler.go#L34